vllm/executor/multiproc_gpu_executor.py

```diff
@@ -122,6 +122,16 @@ def _check_executor_parameters(self):
                 "CUDA_VISIBLE_DEVICES": (",".join(map(str, range(world_size))))
             })
 
+        if (cuda_is_initialized()
+                and os.environ.get("VLLM_WORKER_MULTIPROC_METHOD") != "spawn"):
+            # CUDA was initialized by an application using vllm as a library.
```

Comment:
I disagree with this conclusion. It is one of the possibilities sure, but it is also possible the ordering of operations in `vllmd` has changed given the architectural restructuring that happened in the 0.6 series.

Reply:
Fair enough -- either way, cuda was already initialized, but it may not necessarily be by external code at some point in the future.

Reply:
I removed this comment since the code and log message cover it well enough already.

Reply:
Thanks @russellb .



Providing an example:
# A vllm deployment:
.

If I run something like:
```
```